ando/tools/generator/AnDOGenerator.py

```diff
@@ -26,26 +26,28 @@
 
 
 class AnDOData:
-
+    """
+    Representation of a AnDO Data, as specified by in the [ephys BEP]
+    (https://docs.google.com/document/d/1oG-C8T-dWPqfVzL2W8HO3elWK8NIh2cOCPssRGv23n0/edit#heading=h.7jcxz3flgq5o)
+
+    The AnDOData object can track multiple realizations of `split`, `run`, `task` but only a single
+    realization of `session` and `subject`, i.e. to represent multiple `session` folders, multiple
+    AnDOData objects are required.
+
+    Parameters
+    ----------
+        sub_id : str
+            subject identifier, e.g. '0012' or 'j.s.smith'
+        ses-id : str
+            session identifier, e.g. '2021-01-01' or '007'
+        tasks : list
+            list of strings, the task identifiers used in the session
+        runs : list or dict
+            list of integers, the run identifiers used in the session.
+            In case of more than one task a dictionary needs to be provided with the task as keys
+            and the list of run identifiers as corresponding values
+    """
```

Comment:
```suggestion
    ----------
    sub_id : str
        subject identifier, e.g. '0012' or 'j.s.smith'
    ses-id : str
        session identifier, e.g. '2021-01-01' or '007'
    tasks : list
        list of strings, the task identifiers used in the session
    runs : list or dict
        list of integers, the run identifiers used in the session.
        In case of more than one task a dictionary needs to be provided with the task as keys
        and the list of run identifiers as corresponding values
    """
```